robottelo/hosts.py

```diff
@@ -65,13 +65,13 @@ def get_sat_rhel_version():
     if not available fallback to robottelo configuration."""
 
     try:
-        rhel_version = Satellite().os_version
+        rhel_release = Satellite().os_version
     except (AuthenticationError, ContentHostError, BoxKeyError):
         if hasattr(settings.server.version, 'rhel_release'):
-            rhel_version = str(settings.server.version.rhel_release)
+            rhel_release = str(settings.server.version.rhel_release)
         elif hasattr(settings.robottelo, 'rhel_version'):
-            rhel_version = settings.robottelo.rhel_version
-    return Version(rhel_version)
+            rhel_release = settings.robottelo.rhel_version
```

Comment:
@shweta83 If you're removing the `rhel_version` setting from `server.yaml`, not sure if this elif block will be required

Reply:
> @shweta83 If you're removing the `rhel_version` setting from `server.yaml`, not sure if this elif block will be required

@Gauravtalreja1 elif block gets the rhel version from robottelo.yaml.template in case RHEL_RELEASE is blank in server.yaml.template(If block)

Reply:
@shweta83 yes, but you're already removing `rhel_version` from `server.yaml.template`,  so there will be no `settings.robottelo.rhel_version`, right?

Reply:
ah, my bad, you're referencing `rhel_version` from `robottelo.yaml`, /ignore me :)